src/libcadet/model/LumpedRateModelWithoutPores.cpp

```diff
@@ -199,7 +199,14 @@ bool LumpedRateModelWithoutPores::configureModelDiscretization(IParameterProvide
 	clearBindingModels();
 	_binding.push_back(nullptr);
 
-	_binding[0] = helper.createBindingModel(paramProvider.getString("ADSORPTION_MODEL"));
+	_binding[0] = helper.createBindingModel("NONE");
```

Comment:
Move this line into the `else` clause of the following `if`: If `ADSORPTION_MODEL` is provided, the `NONE`-model created here is not `delete`d causing a memory leak.

Reply:
Done.